classes/auto-package-libs.oeclass

```diff
@@ -172,6 +173,9 @@ def auto_package_libs (d):
         devpkg_provides = (d.get("PROVIDES_" + devpkg) or "").split()
         devpkg_provides.append("%s%s-dev"%(provideprefix,
                                            lib.replace("_", "-").lower()))
+        if library_version:
+            devpkg_provides.append("%s%s-%s-dev"%(
+                provideprefix, lib.replace("_","-").lower(), library_version))
```

Comment:
This part definitely makes sense. Please just tell me if the above are indeed intended as no-op changes, then ok.

Reply:
Exactly.  This is the only real change.  I will rebase into two commits to make it more clear.